[2022-07-29 15:38:00.471] [remoteagent] [info] Extension host agent started.
[2022-07-29 15:38:03.078] [remoteagent] [error] [127.0.0.1][709b5770][ManagementConnection] Unknown reconnection token (never seen).
[2022-07-29 15:38:03.083] [remoteagent] [error] [127.0.0.1][7a58202d][ExtensionHostConnection] Unknown reconnection token (never seen).
[2022-07-29 17:08:17.332] [remoteagent] [info] [127.0.0.1][cd0670fe][ManagementConnection] New connection established.
[2022-07-29 17:08:17.433] [remoteagent] [info] [127.0.0.1][8487b0b7][ExtensionHostConnection] New connection established.
[2022-07-29 17:08:17.436] [remoteagent] [info] [127.0.0.1][8487b0b7][ExtensionHostConnection] <21891> Launched Extension Host Process.
[2022-07-29 17:08:19.670] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateTitle (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-29 17:08:32.483] [remoteagent] [info] [127.0.0.1][8487b0b7][ExtensionHostConnection] <21891> Extension Host Process exited with code: 0, signal: null.
[2022-07-29 17:08:32.484] [remoteagent] [info] Last EH closed, waiting before shutting down
[2022-07-29 17:08:36.465] [remoteagent] [info] Got delay-shutdown request while in shutdown timeout, delaying
[2022-07-29 17:08:36.465] [remoteagent] [info] Cancelling previous shutdown timeout
[2022-07-29 17:08:37.017] [remoteagent] [info] [127.0.0.1][292dbb04][ManagementConnection] New connection established.
[2022-07-29 17:08:37.121] [remoteagent] [info] [127.0.0.1][b4646256][ExtensionHostConnection] New connection established.
[2022-07-29 17:08:37.125] [remoteagent] [info] [127.0.0.1][b4646256][ExtensionHostConnection] <22022> Launched Extension Host Process.
[2022-07-29 17:08:37.568] [remoteagent] [info] [127.0.0.1][cd0670fe][ManagementConnection] The client has disconnected, will wait for reconnection 3h before disposing...
[2022-07-29 17:08:37.575] [remoteagent] [error] Error: Unexpected SIGPIPE
    at process.<anonymous> (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/server/node/server.main.js:145:7933)
    at process.emit (node:events:390:28)
[2022-07-29 17:09:02.898] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-29 17:09:34.062] [remoteagent] [error] No ptyHost heartbeat after 12 seconds
[2022-07-29 17:12:47.344] [remoteagent] [info] [127.0.0.1][b4646256][ExtensionHostConnection] <22022> Extension Host Process exited with code: 0, signal: null.
[2022-07-29 17:12:47.345] [remoteagent] [info] Cancelling previous shutdown timeout
[2022-07-29 17:12:47.345] [remoteagent] [info] Last EH closed, waiting before shutting down
[2022-07-29 17:12:47.354] [remoteagent] [info] [127.0.0.1][292dbb04][ManagementConnection] The client has disconnected, will wait for reconnection 3h before disposing...
[2022-07-29 17:12:47.354] [remoteagent] [error] Error: Unexpected SIGPIPE
    at process.<anonymous> (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/server/node/server.main.js:145:7933)
    at process.emit (node:events:390:28)
[2022-07-29 17:12:48.103] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateIcon (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25409)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-29 17:12:48.104] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateTitle (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-29 17:12:48.104] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateTitle (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-29 17:16:48.647] [remoteagent] [info] Got delay-shutdown request while in shutdown timeout, delaying
[2022-07-29 17:16:48.647] [remoteagent] [info] Cancelling previous shutdown timeout
[2022-07-29 17:16:49.063] [remoteagent] [info] [127.0.0.1][cd0670fe][ManagementConnection] Another client has connected, will shorten the wait for reconnection 5m before disposing...
[2022-07-29 17:16:49.064] [remoteagent] [info] [127.0.0.1][292dbb04][ManagementConnection] Another client has connected, will shorten the wait for reconnection 5m before disposing...
[2022-07-29 17:16:49.064] [remoteagent] [info] [127.0.0.1][55c0d848][ManagementConnection] New connection established.
[2022-07-29 17:16:49.176] [remoteagent] [info] [127.0.0.1][0387379a][ExtensionHostConnection] New connection established.
[2022-07-29 17:16:49.180] [remoteagent] [info] [127.0.0.1][0387379a][ExtensionHostConnection] <22222> Launched Extension Host Process.
[2022-07-29 17:16:51.235] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateTitle (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-29 17:17:18.508] [remoteagent] [info] [127.0.0.1][0387379a][ExtensionHostConnection] <22222> Extension Host Process exited with code: 0, signal: null.
[2022-07-29 17:17:18.509] [remoteagent] [info] Cancelling previous shutdown timeout
[2022-07-29 17:17:18.509] [remoteagent] [info] Last EH closed, waiting before shutting down
[2022-07-29 17:17:21.791] [remoteagent] [info] Got delay-shutdown request while in shutdown timeout, delaying
[2022-07-29 17:17:21.791] [remoteagent] [info] Cancelling previous shutdown timeout
[2022-07-29 17:17:22.307] [remoteagent] [info] [127.0.0.1][24ab53db][ManagementConnection] New connection established.
[2022-07-29 17:17:22.458] [remoteagent] [info] [127.0.0.1][9024b7e9][ExtensionHostConnection] New connection established.
[2022-07-29 17:17:22.484] [remoteagent] [info] [127.0.0.1][9024b7e9][ExtensionHostConnection] <22409> Launched Extension Host Process.
[2022-07-29 17:17:22.993] [remoteagent] [info] [127.0.0.1][55c0d848][ManagementConnection] The client has disconnected, will wait for reconnection 3h before disposing...
[2022-07-29 17:17:22.997] [remoteagent] [error] Error: Unexpected SIGPIPE
    at process.<anonymous> (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/server/node/server.main.js:145:7933)
    at process.emit (node:events:390:28)
[2022-07-29 17:17:24.425] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateTitle (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-29 17:17:46.349] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-29 17:17:58.797] [remoteagent] [error] No ptyHost heartbeat after 12 seconds
[2022-07-29 17:18:37.527] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-29 17:18:55.449] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-29 17:21:19.282] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-29 17:23:11.912] [remoteagent] [info] New EH opened, aborting shutdown
[2022-07-29 17:23:56.894] [remoteagent] [info] [127.0.0.1][24ab53db][ManagementConnection] The client has disconnected, will wait for reconnection 3h before disposing...
[2022-07-29 17:23:57.908] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-29 17:24:00.315] [remoteagent] [error] Error: Unexpected SIGPIPE
    at process.<anonymous> (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/server/node/server.main.js:145:7933)
    at process.emit (node:events:390:28)
[2022-07-29 17:24:01.408] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateIcon (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25409)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-29 17:24:06.722] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateTitle (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-29 17:24:06.742] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateTitle (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-29 17:24:06.965] [remoteagent] [info] [127.0.0.1][9024b7e9][ExtensionHostConnection] <22409> Extension Host Process exited with code: 0, signal: null.
[2022-07-29 17:24:06.965] [remoteagent] [info] Last EH closed, waiting before shutting down
[2022-07-29 17:24:07.364] [remoteagent] [info] Got delay-shutdown request while in shutdown timeout, delaying
[2022-07-29 17:24:07.364] [remoteagent] [info] Cancelling previous shutdown timeout
[2022-07-29 17:24:07.688] [remoteagent] [info] [127.0.0.1][55c0d848][ManagementConnection] Another client has connected, will shorten the wait for reconnection 5m before disposing...
[2022-07-29 17:24:07.688] [remoteagent] [info] [127.0.0.1][24ab53db][ManagementConnection] Another client has connected, will shorten the wait for reconnection 5m before disposing...
[2022-07-29 17:24:07.688] [remoteagent] [info] [127.0.0.1][a687745d][ManagementConnection] New connection established.
[2022-07-29 17:24:07.792] [remoteagent] [info] [127.0.0.1][7df4320d][ExtensionHostConnection] New connection established.
[2022-07-29 17:24:07.797] [remoteagent] [info] [127.0.0.1][7df4320d][ExtensionHostConnection] <22661> Launched Extension Host Process.
[2022-07-29 17:24:09.925] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateTitle (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-29 17:25:15.172] [remoteagent] [info] [127.0.0.1][7df4320d][ExtensionHostConnection] <22661> Extension Host Process exited with code: 0, signal: null.
[2022-07-29 17:25:15.173] [remoteagent] [info] Cancelling previous shutdown timeout
[2022-07-29 17:25:15.173] [remoteagent] [info] Last EH closed, waiting before shutting down
[2022-07-29 17:25:18.835] [remoteagent] [info] Got delay-shutdown request while in shutdown timeout, delaying
[2022-07-29 17:25:18.835] [remoteagent] [info] Cancelling previous shutdown timeout
[2022-07-29 17:25:19.225] [remoteagent] [info] [127.0.0.1][e2e71c9e][ManagementConnection] New connection established.
[2022-07-29 17:25:19.471] [remoteagent] [info] [127.0.0.1][c8489041][ExtensionHostConnection] New connection established.
[2022-07-29 17:25:19.474] [remoteagent] [info] [127.0.0.1][c8489041][ExtensionHostConnection] <22820> Launched Extension Host Process.
[2022-07-29 17:25:20.014] [remoteagent] [info] [127.0.0.1][a687745d][ManagementConnection] The client has disconnected, will wait for reconnection 3h before disposing...
[2022-07-29 17:25:20.021] [remoteagent] [error] Error: Unexpected SIGPIPE
    at process.<anonymous> (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/server/node/server.main.js:145:7933)
    at process.emit (node:events:390:28)
[2022-07-29 17:25:21.673] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateTitle (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-29 17:28:21.349] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-29 17:33:19.671] [remoteagent] [info] New EH opened, aborting shutdown
[2022-07-29 17:33:22.586] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-29 17:34:26.671] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-29 17:34:33.987] [remoteagent] [error] No ptyHost heartbeat after 12 seconds
[2022-07-29 17:34:53.295] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-29 17:35:06.554] [remoteagent] [info] [127.0.0.1][e2e71c9e][ManagementConnection] The client has disconnected, will wait for reconnection 3h before disposing...
[2022-07-29 17:35:08.183] [remoteagent] [error] No ptyHost heartbeat after 12 seconds
[2022-07-29 17:35:30.698] [remoteagent] [error] Error: Unexpected SIGPIPE
    at process.<anonymous> (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/server/node/server.main.js:145:7933)
    at process.emit (node:events:390:28)
[2022-07-29 17:36:59.389] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateIcon (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25409)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-29 17:37:00.054] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateTitle (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-29 17:37:00.374] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateTitle (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-29 17:37:05.080] [remoteagent] [info] [127.0.0.1][c8489041][ExtensionHostConnection] <22820> Extension Host Process exited with code: 0, signal: null.
[2022-07-29 17:37:05.081] [remoteagent] [info] Last EH closed, waiting before shutting down
[2022-07-29 17:37:05.185] [remoteagent] [info] Got delay-shutdown request while in shutdown timeout, delaying
[2022-07-29 17:37:05.185] [remoteagent] [info] Cancelling previous shutdown timeout
[2022-07-29 17:37:05.543] [remoteagent] [info] [127.0.0.1][a687745d][ManagementConnection] Another client has connected, will shorten the wait for reconnection 5m before disposing...
[2022-07-29 17:37:05.545] [remoteagent] [info] [127.0.0.1][e2e71c9e][ManagementConnection] Another client has connected, will shorten the wait for reconnection 5m before disposing...
[2022-07-29 17:37:05.546] [remoteagent] [info] [127.0.0.1][558dcbe9][ManagementConnection] New connection established.
[2022-07-29 17:37:05.640] [remoteagent] [info] [127.0.0.1][9d30ee5c][ExtensionHostConnection] New connection established.
[2022-07-29 17:37:05.646] [remoteagent] [info] [127.0.0.1][9d30ee5c][ExtensionHostConnection] <22988> Launched Extension Host Process.
[2022-07-29 17:37:07.902] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateTitle (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-29 17:38:21.639] [remoteagent] [info] [127.0.0.1][9d30ee5c][ExtensionHostConnection] <22988> Extension Host Process exited with code: 0, signal: null.
[2022-07-29 17:38:21.640] [remoteagent] [info] Cancelling previous shutdown timeout
[2022-07-29 17:38:21.640] [remoteagent] [info] Last EH closed, waiting before shutting down
[2022-07-29 17:38:25.051] [remoteagent] [info] Got delay-shutdown request while in shutdown timeout, delaying
[2022-07-29 17:38:25.051] [remoteagent] [info] Cancelling previous shutdown timeout
[2022-07-29 17:38:25.417] [remoteagent] [info] [127.0.0.1][3115d9ca][ManagementConnection] New connection established.
[2022-07-29 17:38:25.643] [remoteagent] [info] [127.0.0.1][51668f61][ExtensionHostConnection] New connection established.
[2022-07-29 17:38:25.650] [remoteagent] [info] [127.0.0.1][51668f61][ExtensionHostConnection] <23187> Launched Extension Host Process.
[2022-07-29 17:38:26.171] [remoteagent] [info] [127.0.0.1][558dcbe9][ManagementConnection] The client has disconnected, will wait for reconnection 3h before disposing...
[2022-07-29 17:38:26.223] [remoteagent] [error] Error: Unexpected SIGPIPE
    at process.<anonymous> (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/server/node/server.main.js:145:7933)
    at process.emit (node:events:390:28)
[2022-07-29 17:38:27.973] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateTitle (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-29 17:38:41.163] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-29 17:38:49.774] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-29 17:39:01.059] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-29 17:39:26.660] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-29 17:41:07.095] [remoteagent] [error] No ptyHost heartbeat after 12 seconds
[2022-07-29 17:41:52.475] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-29 17:42:18.390] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateIcon (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25409)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-29 17:42:19.008] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateTitle (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-29 17:42:22.310] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateTitle (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-29 17:42:28.410] [remoteagent] [info] [127.0.0.1][cd0670fe][ManagementConnection] The reconnection short grace time of 5m has expired, so the connection will be disposed.
[2022-07-29 17:42:31.077] [remoteagent] [info] [127.0.0.1][292dbb04][ManagementConnection] The reconnection short grace time of 5m has expired, so the connection will be disposed.
[2022-07-29 17:42:39.177] [remoteagent] [info] Got delay-shutdown request while in shutdown timeout, delaying
[2022-07-29 17:42:39.383] [remoteagent] [info] Cancelling previous shutdown timeout
[2022-07-29 17:42:41.289] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateTitle (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-29 17:42:41.409] [remoteagent] [info] [127.0.0.1][558dcbe9][ManagementConnection] Another client has connected, will shorten the wait for reconnection 5m before disposing...
[2022-07-29 17:42:41.841] [remoteagent] [info] [127.0.0.1][3115d9ca][ManagementConnection] The client has reconnected.
[2022-07-29 17:42:42.613] [remoteagent] [info] [127.0.0.1][51668f61][ExtensionHostConnection] The client has reconnected.
[2022-07-29 17:43:44.208] [remoteagent] [info] [127.0.0.1][55c0d848][ManagementConnection] The reconnection short grace time of 5m has expired, so the connection will be disposed.
[2022-07-29 17:43:44.209] [remoteagent] [info] [127.0.0.1][24ab53db][ManagementConnection] The reconnection short grace time of 5m has expired, so the connection will be disposed.
[2022-07-29 17:45:12.457] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-29 17:45:18.780] [remoteagent] [error] No ptyHost heartbeat after 12 seconds
[2022-07-29 17:45:58.894] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-29 17:46:12.968] [remoteagent] [error] No ptyHost heartbeat after 12 seconds
[2022-07-29 17:48:03.204] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-29 17:48:32.987] [remoteagent] [info] New EH opened, aborting shutdown
[2022-07-29 17:51:53.136] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateIcon (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25409)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-29 17:51:53.137] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateTitle (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-29 17:51:53.145] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateTitle (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-29 17:51:54.060] [remoteagent] [info] [127.0.0.1][b424d31b][ManagementConnection] New connection established.
[2022-07-29 17:51:54.185] [remoteagent] [info] [127.0.0.1][bb8b1fe7][ExtensionHostConnection] New connection established.
[2022-07-29 17:51:54.199] [remoteagent] [info] [127.0.0.1][bb8b1fe7][ExtensionHostConnection] <23766> Launched Extension Host Process.
[2022-07-29 17:51:55.021] [remoteagent] [info] [127.0.0.1][3115d9ca][ManagementConnection] The client has disconnected, will wait for reconnection 3h before disposing...
[2022-07-29 17:51:55.021] [remoteagent] [error] Error: Unexpected SIGPIPE
    at process.<anonymous> (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/server/node/server.main.js:145:7933)
    at process.emit (node:events:390:28)
[2022-07-29 17:51:56.041] [remoteagent] [info] [127.0.0.1][a687745d][ManagementConnection] The reconnection short grace time of 5m has expired, so the connection will be disposed.
[2022-07-29 17:51:56.042] [remoteagent] [info] [127.0.0.1][e2e71c9e][ManagementConnection] The reconnection short grace time of 5m has expired, so the connection will be disposed.
[2022-07-29 17:51:56.654] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateTitle (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-29 17:52:10.462] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-29 17:52:48.394] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-29 17:53:25.126] [remoteagent] [info] [127.0.0.1][b424d31b][ManagementConnection] The client has disconnected, will wait for reconnection 3h before disposing...
[2022-07-29 17:53:26.410] [remoteagent] [error] Error: Unexpected SIGPIPE
    at process.<anonymous> (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/server/node/server.main.js:145:7933)
    at process.emit (node:events:390:28)
[2022-07-29 17:53:37.944] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateIcon (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25409)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-29 17:53:39.849] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateTitle (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-29 17:53:41.697] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateTitle (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-29 17:55:42.768] [remoteagent] [info] [127.0.0.1][558dcbe9][ManagementConnection] The reconnection short grace time of 5m has expired, so the connection will be disposed.
[2022-07-29 17:56:54.661] [remoteagent] [info] [127.0.0.1][3115d9ca][ManagementConnection] Another client has connected, will shorten the wait for reconnection 5m before disposing...
[2022-07-29 17:56:54.663] [remoteagent] [info] [127.0.0.1][b424d31b][ManagementConnection] Another client has connected, will shorten the wait for reconnection 5m before disposing...
[2022-07-29 17:56:54.663] [remoteagent] [info] [127.0.0.1][2a5f0170][ManagementConnection] New connection established.
[2022-07-29 17:56:54.762] [remoteagent] [info] [127.0.0.1][f7089931][ExtensionHostConnection] New connection established.
[2022-07-29 17:56:54.771] [remoteagent] [info] [127.0.0.1][f7089931][ExtensionHostConnection] <23944> Launched Extension Host Process.
[2022-07-29 17:57:12.496] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateTitle (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-29 17:57:13.151] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-29 17:58:13.838] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateIcon (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25409)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-29 17:58:13.840] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateTitle (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-29 17:58:13.840] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateTitle (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-29 17:58:14.844] [remoteagent] [info] [127.0.0.1][2a5f0170][ManagementConnection] The client has reconnected.
[2022-07-29 17:59:21.879] [remoteagent] [info] [127.0.0.1][f7089931][ExtensionHostConnection] <23944> Extension Host Process exited with code: 0, signal: null.
[2022-07-29 17:59:26.272] [remoteagent] [info] [127.0.0.1][5761cf7e][ManagementConnection] New connection established.
[2022-07-29 17:59:26.364] [remoteagent] [info] [127.0.0.1][36476c8d][ExtensionHostConnection] New connection established.
[2022-07-29 17:59:26.368] [remoteagent] [info] [127.0.0.1][36476c8d][ExtensionHostConnection] <24233> Launched Extension Host Process.
[2022-07-29 17:59:26.891] [remoteagent] [info] [127.0.0.1][2a5f0170][ManagementConnection] The client has disconnected, will wait for reconnection 3h before disposing...
[2022-07-29 17:59:26.901] [remoteagent] [error] Error: Unexpected SIGPIPE
    at process.<anonymous> (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/server/node/server.main.js:145:7933)
    at process.emit (node:events:390:28)
[2022-07-29 17:59:28.300] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateTitle (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-29 17:59:45.747] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-29 17:59:52.992] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-29 18:00:05.926] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-29 18:00:25.361] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-29 18:00:41.989] [remoteagent] [error] No ptyHost heartbeat after 12 seconds
[2022-07-29 18:03:14.682] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-29 18:03:19.713] [remoteagent] [info] [127.0.0.1][5761cf7e][ManagementConnection] The client has disconnected, will wait for reconnection 3h before disposing...
[2022-07-29 18:03:24.793] [remoteagent] [error] Error: Unexpected SIGPIPE
    at process.<anonymous> (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/server/node/server.main.js:145:7933)
    at process.emit (node:events:390:28)
[2022-07-29 18:03:25.894] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-29 18:03:31.608] [remoteagent] [error] No ptyHost heartbeat after 12 seconds
[2022-07-29 18:03:39.233] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateIcon (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25409)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-29 18:03:43.139] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-29 18:03:45.242] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateTitle (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-29 18:03:47.083] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateTitle (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-29 18:03:49.274] [remoteagent] [error] No ptyHost heartbeat after 12 seconds
[2022-07-29 18:03:52.560] [remoteagent] [info] [127.0.0.1][51668f61][ExtensionHostConnection] <23187> Extension Host Process exited with code: 0, signal: null.
[2022-07-29 18:03:52.618] [remoteagent] [info] [127.0.0.1][36476c8d][ExtensionHostConnection] <24233> Extension Host Process exited with code: 0, signal: null.
[2022-07-29 18:03:53.738] [remoteagent] [info] [127.0.0.1][2a5f0170][ManagementConnection] Another client has connected, will shorten the wait for reconnection 5m before disposing...
[2022-07-29 18:03:53.738] [remoteagent] [info] [127.0.0.1][5761cf7e][ManagementConnection] Another client has connected, will shorten the wait for reconnection 5m before disposing...
[2022-07-29 18:03:53.739] [remoteagent] [info] [127.0.0.1][598c80e2][ManagementConnection] New connection established.
[2022-07-29 18:03:53.822] [remoteagent] [info] [127.0.0.1][d98cd6c0][ExtensionHostConnection] New connection established.
[2022-07-29 18:03:53.827] [remoteagent] [info] [127.0.0.1][d98cd6c0][ExtensionHostConnection] <24495> Launched Extension Host Process.
[2022-07-29 18:03:56.204] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateTitle (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-29 18:04:09.142] [remoteagent] [info] [127.0.0.1][bb8b1fe7][ExtensionHostConnection] <23766> Extension Host Process exited with code: 0, signal: null.
[2022-07-29 18:04:42.170] [remoteagent] [info] [127.0.0.1][d98cd6c0][ExtensionHostConnection] <24495> Extension Host Process exited with code: 0, signal: null.
[2022-07-29 18:04:42.170] [remoteagent] [info] Last EH closed, waiting before shutting down
[2022-07-29 18:04:45.862] [remoteagent] [info] Got delay-shutdown request while in shutdown timeout, delaying
[2022-07-29 18:04:45.862] [remoteagent] [info] Cancelling previous shutdown timeout
[2022-07-29 18:04:46.399] [remoteagent] [info] [127.0.0.1][9caa9444][ManagementConnection] New connection established.
[2022-07-29 18:04:46.563] [remoteagent] [info] [127.0.0.1][4bc5b155][ExtensionHostConnection] New connection established.
[2022-07-29 18:04:46.568] [remoteagent] [info] [127.0.0.1][4bc5b155][ExtensionHostConnection] <24660> Launched Extension Host Process.
[2022-07-29 18:04:46.906] [remoteagent] [info] [127.0.0.1][598c80e2][ManagementConnection] The client has disconnected, will wait for reconnection 3h before disposing...
[2022-07-29 18:04:46.913] [remoteagent] [error] Error: Unexpected SIGPIPE
    at process.<anonymous> (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/server/node/server.main.js:145:7933)
    at process.emit (node:events:390:28)
[2022-07-29 18:06:41.394] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-29 18:08:34.207] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-29 18:09:05.591] [remoteagent] [info] [127.0.0.1][9caa9444][ManagementConnection] The client has disconnected, will wait for reconnection 3h before disposing...
[2022-07-29 18:09:07.410] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-29 18:09:17.672] [remoteagent] [error] Error: Unexpected SIGPIPE
    at process.<anonymous> (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/server/node/server.main.js:145:7933)
    at process.emit (node:events:390:28)
[2022-07-29 18:09:22.795] [remoteagent] [error] No ptyHost heartbeat after 12 seconds
[2022-07-29 18:10:25.389] [remoteagent] [info] New EH opened, aborting shutdown
[2022-07-29 18:12:53.450] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateIcon (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25409)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-29 18:12:58.162] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateTitle (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-29 18:13:03.383] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateTitle (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-29 18:13:17.174] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-29 18:16:57.179] [remoteagent] [info] [127.0.0.1][598c80e2][ManagementConnection] Another client has connected, will shorten the wait for reconnection 5m before disposing...
[2022-07-29 18:16:57.189] [remoteagent] [info] [127.0.0.1][9caa9444][ManagementConnection] Another client has connected, will shorten the wait for reconnection 5m before disposing...
[2022-07-29 18:16:57.192] [remoteagent] [info] [127.0.0.1][926eefa5][ManagementConnection] New connection established.
[2022-07-29 18:16:57.308] [remoteagent] [info] [127.0.0.1][84e30c99][ExtensionHostConnection] New connection established.
[2022-07-29 18:16:57.398] [remoteagent] [info] [127.0.0.1][84e30c99][ExtensionHostConnection] <24943> Launched Extension Host Process.
[2022-07-29 18:19:22.759] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-29 18:19:22.833] [remoteagent] [info] [127.0.0.1][926eefa5][ManagementConnection] The client has disconnected, will wait for reconnection 3h before disposing...
[2022-07-29 18:19:22.847] [remoteagent] [error] Error: Unexpected SIGPIPE
    at process.<anonymous> (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/server/node/server.main.js:145:7933)
    at process.emit (node:events:390:28)
[2022-07-29 18:19:23.901] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateIcon (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25409)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-29 18:19:23.907] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateTitle (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-29 18:19:23.911] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateTitle (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-29 18:19:24.507] [remoteagent] [info] [127.0.0.1][926eefa5][ManagementConnection] Another client has connected, will shorten the wait for reconnection 5m before disposing...
[2022-07-29 18:19:24.508] [remoteagent] [info] [127.0.0.1][d56bc7f0][ManagementConnection] New connection established.
[2022-07-29 18:19:24.634] [remoteagent] [info] [127.0.0.1][1e4bf520][ExtensionHostConnection] New connection established.
[2022-07-29 18:19:24.645] [remoteagent] [info] [127.0.0.1][1e4bf520][ExtensionHostConnection] <25087> Launched Extension Host Process.
[2022-07-29 18:19:26.994] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateTitle (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-29 18:19:52.437] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-29 18:21:03.398] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-29 18:21:37.700] [remoteagent] [info] [127.0.0.1][3115d9ca][ManagementConnection] The reconnection short grace time of 5m has expired, so the connection will be disposed.
[2022-07-29 18:22:02.243] [remoteagent] [info] [127.0.0.1][b424d31b][ManagementConnection] The reconnection short grace time of 5m has expired, so the connection will be disposed.
[2022-07-29 18:22:54.616] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-29 18:23:52.129] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-29 18:24:01.504] [remoteagent] [error] No ptyHost heartbeat after 12 seconds
[2022-07-29 18:24:03.055] [remoteagent] [info] [127.0.0.1][d56bc7f0][ManagementConnection] The client has disconnected, will wait for reconnection 3h before disposing...
[2022-07-29 18:24:04.974] [remoteagent] [error] Error: Unexpected SIGPIPE
    at process.<anonymous> (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/server/node/server.main.js:145:7933)
    at process.emit (node:events:390:28)
[2022-07-29 18:24:08.495] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateIcon (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25409)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-29 18:24:09.758] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateTitle (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-29 18:24:11.274] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateTitle (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-29 18:24:49.387] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-29 18:26:28.738] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-29 18:26:35.777] [remoteagent] [error] No ptyHost heartbeat after 12 seconds
[2022-07-29 18:29:34.954] [remoteagent] [info] [127.0.0.1][2a5f0170][ManagementConnection] The reconnection short grace time of 5m has expired, so the connection will be disposed.
[2022-07-29 18:29:40.915] [remoteagent] [info] [127.0.0.1][5761cf7e][ManagementConnection] The reconnection short grace time of 5m has expired, so the connection will be disposed.
[2022-07-29 18:33:02.315] [remoteagent] [info] [127.0.0.1][598c80e2][ManagementConnection] The reconnection short grace time of 5m has expired, so the connection will be disposed.
[2022-07-29 18:33:04.660] [remoteagent] [info] [127.0.0.1][9caa9444][ManagementConnection] The reconnection short grace time of 5m has expired, so the connection will be disposed.
[2022-07-29 18:33:33.648] [remoteagent] [info] [127.0.0.1][926eefa5][ManagementConnection] The reconnection short grace time of 5m has expired, so the connection will be disposed.
[2022-07-29 18:41:07.742] [remoteagent] [info] [127.0.0.1][1e4bf520][ExtensionHostConnection] <25087> Extension Host Process exited with code: 0, signal: null.
[2022-07-29 18:41:26.910] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-29 18:41:35.292] [remoteagent] [error] No ptyHost heartbeat after 12 seconds
[2022-07-29 18:44:08.746] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-29 18:44:41.526] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-29 18:51:40.519] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
